### PR TITLE
Update adoption-score.md

### PR DESCRIPTION
**Pull request by msbemba**
Adoption score not available in GCC.

per issue #https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9814

No other reference document.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/adoption/adoption-score.md [hunk 1]
```diff
@@ -72,6 +72,9 @@ The role-based access control model for Adoption Score helps organizations furth
 Microsoft is committed to protecting individual privacy. This [privacy document](privacy.md)  explains the controls we provide you, as your organization's IT administrator, to ensure that the information is actionable while not compromising the trust you place in Microsoft.
 
 You can access the experience from Microsoft 365 Admin home under **Reports** > **Adoption Score**.
+
+> [!NOTE]
+> This feature is not available in GCC High, GCC and DOD tenants.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/adoption/adoption-score.md]:
```suggestion
> This feature is not available in GCC High, GCC, and DOD tenants.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@efrene Please merge this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@efrene Gentle reminder.

**Pull request merged**